frontend/src/host_orchestrator/orchestrator/createcvdaction.go

```diff
@@ -104,11 +108,24 @@ func (a *CreateCVDAction) launchWithCanonicalConfig(op apiv1.Operation) (*apiv1.
 	if err != nil {
 		return nil, err
 	}
-	log.Printf("environment config:\n%s", string(data))
-	data = bytes.ReplaceAll(data,
-		[]byte(apiv1.EnvConfigUserArtifactsVar+"/"),
-		[]byte(a.userArtifactsDirResolver.GetDirPath("")+"/"))
-	configFile, err := createTempFile("cvdload*.json", data, 0640)
+	config := string(data)
+	log.Printf("environment config:\n%s", config)
+	regex, err := regexp.Compile("\"" + apiv1.EnvConfigUserArtifactsVarV1 + "/[0-9a-zA-Z_,]+\"")
+	if err != nil {
+		return nil, fmt.Errorf("failed to compile regex: %w", err)
+	}
+	var errs *multierror.Error
+	config = regex.ReplaceAllStringFunc(config, func(base string) string {
+		checksums := strings.Split(strings.TrimPrefix(strings.Trim(base, "\""), apiv1.EnvConfigUserArtifactsVarV1), ",")
```

Comment:
I think this leaves the first checksum with a starting `'/'` while the other ones don't have it. It's easier to just trim it here before splitting than to deal with the differences later on.